src/offlinelogstorage/dlt_offline_logstorage.c

```diff
@@ -1052,6 +1058,21 @@ DLT_STATIC int dlt_logstorage_check_nofiles(DltLogStorageFilterConfig *config,
     return dlt_logstorage_read_number(&config->num_files, value);
 }
 
+DLT_STATIC int dlt_logstorage_check_gzip_compression(DltLogStorageFilterConfig *config,
+                                                     char *value)
+{
+#ifdef DLT_LOGSTORAGE_USE_GZIP
+    if ((config == NULL) || (value == NULL))
+        return -1;
+
+    return dlt_logstorage_read_number(&config->gzip_compression, value);
+#else
+    dlt_log(LOG_WARNING, "dlt-daemon not compiled with logstorage gzip support\n");
```

Comment:
I do not refer a warning here, I think LOG_NOTICE or LOG_INFO is enough.

Reply:
All other instances of config flags omitted and similar use warnings hence I chose that. But in this particular case I really think it deserves a warning.

You don't agree that a user trying to enable something through config that isn't available merits a warning? The user is actively attempting to enable a feature that is not available.

Reply:
OK, I agree. It sounds reasonable to me now :)